NEW: Added achievable frequency and latency (processing delay) to Input Debugger (ISX-2254) (#2142)
Unity-Technologies/InputSystem: Packages/com.unity.inputsystem/Documentation~/Debugging.md

![Device in Input Debugger](Images/DeviceInDebugger.png)

> The top of the Device window displays general information about the specific Device, such as name, manufacturer, associated layout, device flags, device ID and serial number.  In addition, this section also display the current __sample frequency__ and __processing delay__ of the deivce.

Reviewer: ```suggestion
The top of the Device window displays general information about the specific Device, such as name, manufacturer, associated layout, device flags, device ID and serial number. This section also displays the current __sample frequency__ and __processing delay__ of the device.
```

> __Sample frequency__ indicates the current frequency of samples or events the Input System is currently processing at given in Hertz (Hz). For devices receiving events this reflects the current flow of events received by the system, and for device receiving periodic readings this reflects the achievable sample rate of the system. The latter may be compared to the globally configured target sampling frequency, while achievable event frequency is uncorrelated to the sample polling frequency setting.

Reviewer: ```suggestion
__Sample frequency__ indicates the frequency in Hertz (Hz) at which the Input System is currently processing samples or events. For devices receiving events this reflects the current flow of events received by the system. For devices receiving periodic readings this reflects the achievable sample rate of the system. The latter may be compared to the globally configured target sampling frequency, while achievable event frequency is uncorrelated to the sample polling frequency setting.
```

Author: Thanks @chrstphrsxtn for the suggestions, its much better.

> __Processing delay__ indicates the average, minimum and maximum latency contribution from creating an input event or reading until the Input System has processed the same input event. Note that this excludes any additional input delay caused by OS, drivers or device communication. Also note that this excludes any additional output latency that may be caused by additional processing, rendering, GPU swap-chains or display refresh rate.

Reviewer: ```suggestion
__Processing delay__ indicates the average, minimum, and maximum time delay from when an input event or reading is created until the Input System has processed it. Note that this excludes any additional input delay caused by OS, drivers, or device communication. Also note that this excludes any additional output latency that may be caused by additional processing, rendering, GPU swap-chains, or display refresh rate.
```
